python-sdk/tutorials/automl-with-azureml/image-classification-multilabel/auto-ml-image-classification-multilabel.ipynb

```diff
@@ -713,13 +855,96 @@
     "        label_offset_y += 30\n",
     "plt.show()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Generate Explanations\n",
+    "- Explainability methods:\n",
+    "   - xrai\n",
+    "   - integratedgradients\n",
+    "   - guidedgradcam\n",
```

Comment:
nit: should we say the full name for the customer, and then also give the code / enum value? Wondering if we can make it easier for customer to parse guidedgradcam into three separate words. Maybe not -- just thinking out loud

Reply:
added full names and links